internal/util/util.go

```diff
@@ -149,11 +149,14 @@ type Weber interface {
 	OpenURL(url string)
 }
 
+// Durationer is an interface for exanding strings into times.
```

Comment:
way to come back and document things you're not even modifying 

You get the boy scout award for this PR 🥇

Reply:
👍